### DISCUSSION TITLE
Logstash Syslog Output

### DISCUSSION THREAD

**Discussion by nilssachs** [Security-Onion-Solutions/securityonion#11077] [2.4]
Hey!
I am trying to implement syslog output within SO2.4 and tried it sticking to the manual as described here:
https://docs.securityonion.net/en/2.4/logstash.html#logstash
Unfortunately, I was not able to find neither the manager.sls, nor search.sls. Where would be te right place to include the output config?
Thanks, Nils

**Comment by nilssachs** [comment]
Just realized there was a change in the documentation in the meantime. 
Is a syslog output still possible?

**Comment by defensivedepth** [comment]
What are you attempting to do? Send certain logs from Security Onion to another system via syslog?

**Reply by nilssachs** [reply]
Hi Josh, yes, thats the idea.
Thanks, Nils

**Reply by nilssachs** [reply]
Any update?
Thanks!

**Reply by weslambert** [reply]
You can read how to forward syslog here:

https://docs.securityonion.net/en/2.4/logstash.html#forwarding-events-to-an-external-destination
